### DISCUSSION TITLE
Welcome to essential-ui-kit-for-xamarin.forms Discussions!

### DISCUSSION THREAD

**Discussion by vijays-git** [syncfusion/essential-ui-kit-for-xamarin.forms#68] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jsnsurya** [comment]
It says free but when I installed it asks for licensing key how do i obtain key.

**Reply by jsnsurya** [reply]
About
Free and beautiful XAML template pages for Xamarin.Forms apps.
